spec/cli_spec.rb

```diff
@@ -77,8 +77,8 @@ def test_file(file_name)
         Then{expect(output).to include(args[1])}
       end
 
-      context "when given file has Irresponsible smell" do
```

Comment:
This change can be reverted as well, there is already a spec for non-smelly file [here](https://github.com/ralfclaassens/preek/blob/update_reek_version/spec/cli_spec.rb#L66).